applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -124,6 +124,13 @@ def __init__(self, main_model_part, custom_settings):
         else:
             self.main_model_part.ProcessInfo[KratosMultiphysics.IS_RESTARTED] = False
 
+        # This is needed when the solving functions are called, only NewtonRaphson properly implements
```

Comment:
we should do the change in the linear_strategy too...

Reply:
absolutely, this is why I opened #1911